master/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -46,6 +46,11 @@ kubectl expose --namespace=advanced-policy-demo deployment nginx --port=80
 
 #### Check using calicoctl
 
+  > **Note:**
+  >
+  > This requires the [calicoctl tool to be configured]({{site.baseurl}}/{{page.version}}/reference/calicoctl/setup/etcdv2).
+  > For example: `export ETCD_ENDPOINTS=http://10.96.232.136:6666`
```

Comment:
We could theoretically convert this tutorial to use a self-hosted calicoctl, which I think would be the most "generally applicable" way to do this.